lib/Runtime/Types/DynamicObject.h

```diff
@@ -67,15 +67,8 @@ namespace Js
 
         friend class JavascriptArray; // for xplat offsetof field access
         friend class JavascriptNativeArray; // for xplat offsetof field access
-        friend class JavascriptOperators; // for ReplaceType
```

Comment:
Looks like the xplat builds are failing because `JavascriptOperators` is trying to access a protected member function `DynamicObject::BoxStackInstance`, so this friend wasn't just for ReplaceType

Reply:
True. So the different compilers are enforcing different rules?